source/_integrations/cloudflare.markdown

```diff
@@ -13,87 +13,32 @@ ha_config_flow: true
 ha_integration_type: integration
 ---
 
-With the `cloudflare` integration, you can keep your Cloudflare records up to date.
+With the `cloudflare` integration, you can keep your Cloudflare DNS records up to date.
 
-The integration runs every hour, but can also be started manually by using the service `cloudflare.update_records` under services.
-
-<div class='note warning'>
-
-Due to a limitation in the Cloudflare API, you can not use this integration with any of the following TLD's:
-
-- `.cf`
-- `.ga`
-- `.gq`
-- `.ml`
-- `.tk`
-
-</div>
+The integration runs every hour, but can also be triggered by running the {% my developer_services title="`cloudflare.update_records` service" service="cloudflare.update_records" %}.
 
 ## Requirements
 
-<div class='note warning'>
-
-On older versions of Home Assistant, this integration used the account email and Global API Key to authenticate.
-
-As of Home Assistant 0.117, API Tokens are now used to authenticate.
+The setup requires an API Token created with `Zone:Zone:Read` and `Zone:DNS:Edit` permissions for all zones in your account.
 
-</div>
-
-Setup requires an API Token created with Zone:Zone:Read and Zone:DNS:Edit permissions for all zones in your account.
-
-An easy way to create this is to start with the "Edit zone DNS" template then add Zone:Zone:Read to the permissions.
+An easy way to create this is to start with the "Edit zone DNS" template then add `Zone:Zone:Read` to the permissions.
 
 [Cloudflare API Tokens Guide](https://developers.cloudflare.com/api/tokens/create)
 
 {% include integrations/config_flow.md %}
 
-## Additional information
-
-### Usage of external service
-
-This platform uses the API from [ipify.org](https://www.ipify.org/) to set the public IP address.
+## Limitations
 
-### Home Assistant Companion App
+### Unusable TLDs
 
-If you would like to use [iOS App](https://companion.home-assistant.io/) via Cloudflare set **Minimum TLS version as 1.2**, in order to do that, do the following:
-
-1. Login to your [Cloudflare](https://dash.cloudflare.com/) account.
-2. Choose your domain.
-3. Click on the `SSL/TLS` icon.
-4. Go to tab `Edge Certificates`.
-5. Find `Minimum TLS Version` and set it to **1.2**.
-
-Other settings should not cause any issues.
-
-### SSH over Cloudflare
```

Comment:
While this information can be useful, it's not related to what this integration does.